scripts/install-vsix-dependencies.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const shell = require('shelljs');
+
+// Installs a list of extensions passed on the command line
+
+const testRunFolder = '.vscode-test';
+const testRunFolderAbsolute = path.join(process.cwd(), testRunFolder);
+
+const version = process.env.CODE_VERSION || '*';
+
+const downloadPlatform =
+  process.platform === 'darwin'
+    ? 'darwin'
+    : process.platform === 'win32' ? 'win32-archive' : 'linux-x64';
+
+const windowsExecutable = path.join(testRunFolderAbsolute, 'Code');
+const darwinExecutable = path.join(
```

Comment:
This is basically from https://github.com/Microsoft/vscode-extension-vscode/blob/master/bin/test but modified for OS X since we you cannot just invoke Electron. You need to find the path to the actual code binary to pass the `install-extensions` argument to it.